src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -32,38 +32,46 @@
     WHich can be used for retrieving download links for satellite images for a parameter georegion.
     Or cropping satellite images for the parameter georegion.
    
-    Author: Michael de Winter
+    Author: Michael de Winter, Pieter Kouyzer
 """
 
 
 def correct_file_path(path):
-    "File path does not need to end with /"
+    """
+    Method to check if all file path slashes are correct.
+    Windows file paths can return not correct file paths.
```

Comment:
I think this is fine for now, but it is best to use os.path for file path management. (and even raw strings would be better than this constant changing of path names):

Source:
https://stackoverflow.com/questions/2953834/how-should-i-write-a-windows-path-in-a-python-string-literal

Reply:
when searching for files with glob. it gives the error to sometimes flip one \ into a /, rather annoying

Reply:
Also with the raw strings? Bugger.

Reply:
yes